### DISCUSSION TITLE
Can I install milvus in an ubuntu docker container？

### DISCUSSION THREAD

**Discussion by zlw18813081566** [milvus-io/milvus#28616] [Q&A and General discussion]
我希望用一个镜像完成服务部署，可以将milvus安装到一个ubuntu作为基础镜像的docker容器中吗？

**Comment by yhmo** [comment]
Yes, the official Milvus docker image is made from Ubuntu 20.04.

**Reply by zlw18813081566** [reply]
我只能使用在一个docker容器里安装docker和docker-compose的方式来部署吗？有其他安装方法吗？

**Reply by yhmo** [reply]
milvus-lite可以用pip安装，安装之后是在本地起一个milvus服务进程，一般用来做验证或者测试，不建议用在生产里
https://github.com/milvus-io/milvus-lite

或者在ubuntu/Mac下编译源代码，也可从本地启动milvus进程。
